framework/helpers/BaseConsole.php

```diff
@@ -340,6 +340,93 @@ public static function ansiStrlen($string)
         return mb_strlen(static::stripAnsiFormat($string));
     }
 
+    //
+    // Error code constant names, values, and descriptions are taken from
+    // OpenBSD's sysexits(3) man page http://man.openbsd.org/sysexits
```

Comment:
Could you please elaborate why would we use platform-specific constants here? Do they have same values for say `Windows` or `Linux`?